app/views/shared/_manage_school.html.erb

```diff
@@ -1,31 +1,77 @@
 <li class="nav-item dropdown">
-  <a class="nav-link dropdown-toggle" data-toggle="dropdown" id="manage_school" href="#" role="button" aria-haspopup="true" aria-expanded="false"><%= t('manage_school_menu.manage_school') %></a>
+  <a class="nav-link dropdown-toggle" data-toggle="dropdown" id="manage_school" href="#"
+     role="button" aria-haspopup="true" aria-expanded="false"><%= t('manage_school_menu.manage_school') %></a>
   <div class="dropdown-menu scrollable" aria-labelledby="manage_school" id="manage_school_menu">
     <%= link_to t('manage_school_menu.edit_school_details'), edit_school_path(school), class: 'dropdown-item' %>
-    <%= link_to t('manage_school_menu.edit_school_times'), edit_school_times_path(school), class: 'dropdown-item' if can? :manage_school_times, school %>
-    <%= link_to t('manage_school_menu.your_school_estate'), edit_school_your_school_estate_path(school), class: 'dropdown-item' if EnergySparks::FeatureFlags.active?(:your_school_estates) %>
-    <%= link_to t('manage_school_menu.school_calendar'), calendar_path(school.calendar), class: 'dropdown-item' if school.calendar && can?(:show, school.calendar) %>
-    <%= link_to t('manage_school_menu.manage_users'), school_users_path(school), class: 'dropdown-item' if can? :manage_users, school %>
-    <%= link_to t('manage_school_menu.manage_alert_contacts'), school_contacts_path(school), class: 'dropdown-item' if can? :manage, Contact %>
-    <%= link_to t('manage_school_menu.manage_meters'), school_meters_path(school), class: 'dropdown-item' if can? :index, Meter %>
-    <%= link_to t('manage_school_menu.manage_tariffs'), school_energy_tariffs_path(school), class: 'dropdown-item' if can? :manage, EnergyTariff %>
-    <%= link_to t('manage_school_menu.manage_usage_estimate'), school_estimated_annual_consumptions_path(school), class: 'dropdown-item' if school.latest_annual_estimate.present? || school.suggest_annual_estimate? %>
+    <%= if can? :manage_school_times, school
```

Comment:
Just wondered if you switched these round to please erblint?

Reply:
Yeah this was all just erblint autocorrect